addons/resource/tests/test_resource.py

```diff
@@ -397,40 +329,36 @@ def test_40_calendar_hours_scheduling(self):
-            default_interval=(8, 16), context=context)
+            compute_leaves=True, resource_id=self.resource1.id,
+            default_interval=(8, 16))
         self.assertEqual(res, 32.0, 'resource_calendar: wrong get_working_hours computation')
 
     def test_50_calendar_schedule_days(self):
         """ Testing calendar days scheduling """
-        cr, uid = self.cr, self.uid
-        _format = '%Y-%m-%d %H:%M:%S'
 
         # --------------------------------------------------
         # Test1: with calendar
         # --------------------------------------------------
 
-        res = self.resource_calendar.schedule_days_get_date(cr, uid, self.calendar_id, 5, day_date=self.date1, context={'tz': 'UTC'})
-        self.assertEqual(res.date(), datetime.strptime('2013-02-26 00:0:00', _format).date(), 'resource_calendar: wrong days scheduling')
-        res = self.resource_calendar.schedule_days_get_date(cr, uid, self.calendar_id, -2, day_date=self.date1, context={'tz': 'UTC'})
-        self.assertEqual(res.date(), datetime.strptime('2013-02-08 00:00:00', _format).date(), 'resource_calendar: wrong days scheduling')
+        res = self.calendar.schedule_days_get_date(5, day_date=self.date1)
+        self.assertEqual(res.date(), fields.Datetime.from_string('2013-02-26 00:0:00').date(), 'resource_calendar: wrong days scheduling')
+        res = self.calendar.schedule_days_get_date(-2, day_date=self.date1)
+        self.assertEqual(res.date(), fields.Datetime.from_string('2013-02-08 00:00:00').date(), 'resource_calendar: wrong days scheduling')
 
-        res = self.resource_calendar.schedule_days_get_date(
-            cr, uid, self.calendar_id, 5, day_date=self.date1,
-            compute_leaves=True, resource_id=self.resource1_id, context={'tz': 'UTC'})
-        self.assertEqual(res.date(), datetime.strptime('2013-03-01 00:0:00', _format).date(), 'resource_calendar: wrong days scheduling')
+        res = self.calendar.schedule_days_get_date(5, day_date=self.date1, compute_leaves=True, resource_id=self.resource1.id)
+        self.assertEqual(res.date(), fields.Datetime.from_string('2013-03-01 00:0:00').date(), 'resource_calendar: wrong days scheduling')
 
         # --------------------------------------------------
         # Test2: misc
         # --------------------------------------------------
 
         # Without calendar, should only count days -> 12 -> 16, 5 days with default intervals
-        res = self.resource_calendar.schedule_days_get_date(cr, uid, None, 5, day_date=self.date1, default_interval=(8, 16), context={'tz': 'UTC'})
-        self.assertEqual(res, datetime.strptime('2013-02-16 16:00:00', _format), 'resource_calendar: wrong days scheduling')
+
+        res = self.ResourceCalendar.schedule_days_get_date(5, day_date=self.date1, default_interval=(8, 16))
+        self.assertEqual(res, fields.Datetime.from_string('2013-02-16 16:00:00'), 'resource_calendar: wrong days scheduling')
 
 def seconds(td):
```

Comment:
Global method should be placed after import statements. Also lint the code of this method.